### ISSUE TITLE
Does it work with Vulkan?

### ISSUE THREAD

**Issue by mgateno** [waynepiekarski/XTextureExtractor#18]
Hello Wayne

Have you tested/ported this to Vulkan or is it still only OpenGL?

**Comment by waynepiekarski** [comment]
While X-Plane is now implemented with Vulkan, it still provides OpenGL API support for plugins to continue working. Almost everything is still written in OpenGL, so its not a problem.

However, there is a catch. In XTextureExtractor 1.3, you needed to click a button >> to "find" the right texture, since it could never be detected automatically. In v2.0, I managed to implement an automatic algorithm that always captured the right texture. However, v2.0 doesn't work on Vulkan for reasons I don't understand. But the older v1.3 release with the manual finding algorithm works fine even with XP11 and Vulkan.

So the Vulkan support in v1.3 just works with Vulkan, even though it was written before Vulkan support even existed. Ideally I would like to get the automatic finding algorithm working with Vulkan, but I really would benefit from getting a real API hook by the X-Plane developers to do this properly.
